getCalspec/getCalspec.py

```diff
@@ -137,32 +138,25 @@ def _santiseName(self, name):
             name = 'sdssj151421'
         return name
 
-    def get_spectrum_fits_filename(self, output_directory=None):
-        """
+    def get_spectrum_fits_filename(self):
+        """Downloads the data or pulls it from the cache if available.
+
         Examples
         --------
         >>> c = Calspec("eta1 dor")
-        >>> c.get_spectrum_fits_filename(output_directory="./calspec")
-        './calspec/eta1dor_stis_002.fits'
+        >>> c.get_spectrum_fits_filename()  #doctest: +ELLIPSIS
+        '...astropy/cache/download/url/...'
 
         """
-        if not output_directory:
-            output_directory = os.path.join(_getPackageDir(), "../calspec_data")
-
-        if not os.path.isdir(output_directory):
-            os.mkdir(output_directory)
-
         spectrum_file_name = self._santiseName(self.Name) + self.STIS.replace('*', '') + ".fits"
-        output_file_name = os.path.join(output_directory, spectrum_file_name)
-        if not os.path.isfile(output_file_name):
-            url = CALSPEC_ARCHIVE+spectrum_file_name
-            try:
-                request.urlretrieve(url, output_file_name)
-            except HTTPError as e:
-                raise RuntimeError(f"Failed to get data for {self.Name} from {url}") from e
+        url = CALSPEC_ARCHIVE+spectrum_file_name
```

Comment:
```suggestion
        url = CALSPEC_ARCHIVE + spectrum_file_name
```
Spaces around (non-multiplicative) binary operator.